### PR TITLE
Center console refactor

### PR DESCRIPTION
**Pull request by jonathanAgarcia**
Hey team. here is my PR in which I updated my component to use db data.

### CODE REVIEW

File: front-end/src/components/RightToolBar/FriendsList.js [hunk 1]
```diff
@@ -2,7 +2,15 @@ import React, { useState } from 'react';
 import { Grid, Container, Paper, IconButton } from '@material-ui/core';
 import IndividualFriend from './IndividualFriend.js';
 
-const FriendsList = ({ friendsList, friends }) => {
+const FriendsList = ({ friends, userInput }) => {
```
Comment (chaomonica) [inline front-end/src/components/RightToolBar/FriendsList.js:5]: is userInput used?

File: front-end/src/components/RightToolBar/IndividualFriend.js [hunk 2]
```diff
@@ -3,7 +3,9 @@ import { Grid, Container, Paper, IconButton } from '@material-ui/core';
 import Avatar from '@material-ui/core/Avatar';
 import {Link} from 'react-router-dom';
 
+
 const IndividualFriend = (props) => {
+
   //userId for a friend will be in the original user's object received from the GET request
```
Comment (chaomonica) [inline front-end/src/components/RightToolBar/IndividualFriend.js:9]: haha, feel free to delete the comments :P

**Review by chaomonica — APPROVED** [review]

File: front-end/src/components/RightToolBar/FriendsList.js (hunk 1 above)
Comment (jonathanAgarcia) [reply front-end/src/components/RightToolBar/FriendsList.js:5]: Hey Monica. not currently, but i'm in the process of setting up the use of it now.

File: front-end/src/components/RightToolBar/IndividualFriend.js (hunk 2 above)
Comment (jonathanAgarcia) [reply front-end/src/components/RightToolBar/IndividualFriend.js:9]: Will Do. Thanks

**Pull request merged**